exercises/prime-factors/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "for": {
+    "description": "returns prime factors for the given input number",
```

Comment:
What are these 2 lines trying to achieve?

It's better to stick with the standard format. Have a look at some of the [recent files added by @petertseng](https://github.com/exercism/x-common/search?utf8=%E2%9C%93&q=author%3Apetertseng+type%3Apull+canonical&type=Issues) for good examples of what to do.